canonical-web-and-design/canonical.com
# Bump cookie policy to  v3.03

## Done

- Bump cookie policy to  v3.03
- Update GTM to track initial pageview

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8002/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that you are using cookie policy 3.03
